appcelerator/titanium_mobile
# chore(android): update "titanium-verify" for 9.0.0

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-27738

**Summary:**
- Replaced JAR that was built against 9.0.0 via PR [here](https://github.com/appcelerator/titanium_verify/pull/29).
- Removed its C/C++ *.so libraries.
  * No longer used to decrypt JS files. We now use "ti.cloak" instead.
  * Was missing an `x86_64` `*.so` library, causing an exception to be logged on x86_64 emulator. (Was not crashing. Was safely caught, but it looked bad.)

**Test:**
1. Build [kitchensink-v2](https://github.com/appcelerator/kitchensink-v2) to an `x86_64` emulator.
2. Verify that the app launches and runs fine in the emulator.
3. Check the log. Verify that you do **NOT** see the following error message.
`[ERROR] TiVerify: java.lang.UnsatisfiedLinkError`
4. Build and run to an Android device.
5. Verify that the app launches and runs fine.
